{
    "data_folder_path": "data/CVE-2021-44657",
    "pull_requests_urls": [
        "https://github.com/StackStorm/st2/pull/5359"
    ],
    "cve_name": "CVE-2021-44657",
    "repo": "StackStorm/st2",
    "warnings": {
        "cve": "CVE-2021-44657",
        "warnings": []
    },
    "files_extension_stats": {
        ".txt": 6,
        ".rst": 1,
        ".py": 4
    },
    "file_number": 11,
    "removed_files_number": 0,
    "added_files_number": 0,
    "nb_lines_affected": 46,
    "private_key_files": [],
    "languages": [
        "Python"
    ]
}